com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java

```diff
@@ -433,5 +473,14 @@ public static void create(Project project, File bundleDir, ICanceled canceled) t
 
 		// STEP 5. Use bundletool to create AAB from base.zip
 		File baseAab = createBundle(project, appDir, baseZip, canceled);
+
+		// STEP 6. Copy debug symbols
+		final boolean has_symbols = project.hasOption("with-symbols");
+		if (has_symbols) {
+			copySymbols(project, appDir, title);
+		}
```

Comment:
We take into account `has_symbols` flag and copy-paste debug symbols into the bundle folder.